lms/static/scripts/frontend_apps/components/dashboard/test/OrganizationActivityFilters-test.js

```diff
@@ -0,0 +1,272 @@
+import { MultiSelect } from '@hypothesis/frontend-shared';
+import {
+  checkAccessibility,
+  mockImportedComponents,
+} from '@hypothesis/frontend-testing';
+import { mount } from 'enzyme';
+import sinon from 'sinon';
+
+import { Config } from '../../../config';
+import OrganizationActivityFilters, {
+  $imports,
+} from '../OrganizationActivityFilters';
+
+describe('OrganizationActivityFilters', () => {
+  const courses = [
+    {
+      id: 1,
+      title: 'Course A',
+    },
+    {
+      id: 2,
+      title: 'Course B',
+    },
+  ];
+  const assignments = [
+    {
+      id: 1,
+      title: 'Assignment 1',
+    },
+    {
+      id: 2,
+      title: 'Assignment 2',
+    },
+  ];
+  const students = [
+    {
+      lms_id: '1',
+      display_name: 'First student',
+    },
+    {
+      lms_id: '2',
+      display_name: 'Second student',
+    },
+  ];
+
+  let fakeUseAPIFetch;
+  let fakeConfig;
+  let onCoursesChange;
+  let onAssignmentsChange;
+  let onStudentsChange;
+  let wrappers = [];
+
+  /**
+   * @param {boolean} loading
+   */
+  function configureFakeAPIFetch(fakeUseAPIFetch, loading) {
```

Comment:
Having the second argument to the configuration be an object with a `{ loading: boolean }` shape would make for more self-explanatory call-sites.